docs/examples/nginx/templates/tests/service-test.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
     app.kubernetes.io/name: {{ template "nginx.name" . }}
   annotations:
-    "helm.sh/hook": test-success
+    "helm.sh/test-expect-success": "true"
```

Comment:
If we are to make a breaking change, I'd like to eliminate failure test cases. In practice they are hardly used -- on github it appears that there are about [20 usages of test-failure](https://github.com/search?l=YAML&q=helm.sh%2Fhook+test-failure&type=Code) and [8000 usages of test-success](https://github.com/search?l=YAML&q=helm.sh%2Fhook+test-success&type=Code) (0 and 34 respectively in helm/charts). Where `test-failure` _is_ used it can be replaced by a small amount of logic in the test itself (and this is a better fit since there are a variety of ways a pod can fail aside from what runs in its entrypoint).